judge/views/contests.py

```diff
@@ -149,7 +149,7 @@ def get_object(self, queryset=None):
         user = self.request.user
         profile = self.request.user.profile if user.is_authenticated else None
 
-        if (profile is not None and
+        if (profile is not None and not profile.is_unlisted and
```

Comment:
Unlisted users cannot exist contests that have been privated after their joined?